tests/h/services/rename_user_test.py

```diff
@@ -123,7 +123,10 @@ def test_it_indexes_the_given_ids(self, req, index):
         indexer([1, 2, 3])
 
         batch_indexer = index.BatchIndexer.return_value
-        batch_indexer.index.assert_called_once_with([1, 2, 3])
+        assert 2 == batch_indexer.index.call_count
```

Comment:
Nitpick - Boolean expressions in assert statements are usually written with the actual value on the left and the expected value on the right.

Reply:
fixed.